### ISSUE TITLE
Hide level=true areas in Quests page

### ISSUE THREAD

**Issue by Axiangel** [Map-A-Droid/MAD#578]
Currently all pokestop areas are visible on Quests page, which also includes areas designated for leveling mode.
Filtering out areas with level=true would effective hide all areas designated for level mode from the list in Quests page which would be appreciated on public Quests page.

**Comment by ColdShadow80** [comment]
I do not believe it uses Area name for that filter, since my system has Area names on that page that do not exist in Area setup.
My guess is that is uses Fence name as filter from loaded areas, so it might need additional tweaking in case a fence is both used in a "level=true" area and a "level=false" area.
